File: backend/SmakApi/Models/DTOs/UserInfoDto.cs
namespace SmakApi.Models.DTOs;

public class UserInfoDto
{
    public Guid Id { get; set; }
    public string Name { get; set; } = null!;
    public string Email { get; set; } = null!;
    public string? AvatarUrl { get; set; }
    public string Role { get; set; } = null!;
}

File: mobile/app/src/main/java/com/itstep/smakmobile/dto/UserInfoDto.java
package com.itstep.smakmobile.dto;

import java.util.UUID;

public class UserInfoDto
{
    public UUID id;
    public String name;
    public String email;
    public String avatarUrl;
    public String role;

    public UUID getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public String getRole() {
        return role;
    }
}
